shortnames.conf

```diff
@@ -54,6 +54,9 @@
   "ubi8/ubi-minimal" = "registry.access.redhat.com/ubi8-minimal"
   "ubi8/ubi-init" = "registry.access.redhat.com/ubi8-init"
   "ubi8/ubi-micro" = "registry.access.redhat.com/ubi8-micro"
+  # Rocky Linux
+  "rockylinux" = "docker.io/library/rockylinux"
+  "rocky" = "docker.io/library/rockylinux"
```

Comment:
I am not sure about `rocky` since it's a different name but let @fatherlinux and @rhatdan weigh in.

Reply:
> Yeah, I'd probably keep them the same. Or add two entries.

Sure, i'll remove the `rocky` tag for now. 

Thanks!